CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+# Contributing
+
+If you are reading this, we thank you in advance for willing to contribute to the Watson project! You are awesome.
+
+Before starting to contribute, please read the following:
+
+* [How to contribute](https://tailordev.github.io/Watson/contributing/how-to/)
+* [How to start hacking](https://tailordev.github.io/Watson/contributing/hack/)
+* [Pull request guidelines](https://tailordev.github.io/Watson/contributing/pr-guidelines/)
+* [Contributor Code of Conduct](https://tailordev.github.io/Watson/contributing/coc/)
```

Comment:
Yes, but users browsing our repository and alerted about CONTRIBUTING instructions won't.